best-practices/rails/routes/README.md

```diff
@@ -0,0 +1,19 @@
+## Ruby on Rails Routes
+
+###### RESTful
+
+* Use `resources` to use RESTful routes
+
+* Avoid `member` and `collection` routes.
```

Comment:
I don't have a opinion about it, but i found an best-practice: https://github.com/thoughtbot/guides/tree/master/style/rails